src/Resources/config/routing.yaml

```diff
@@ -0,0 +1,5 @@
+setono_sylius_quickpay_notify:
```

Comment:
Where does Sylius / Payum use this route?

Reply:
![image](https://user-images.githubusercontent.com/7114562/105700060-fcc7cc80-5f10-11eb-8ef7-976169e3f6fa.png)

Reply:
I don't know if it can be set automatically, will check.
If not, then maybe add some info in admin panel.

Reply:
You can set the notify URL on each request. It would be better to have it there if possible

Reply:
This one is at the merchant level, the default, I can't set it on each request.
I was thinking of setting it when the gateway config is updated.
Also, make it optional, with a flag like "accept notification" and show the url.
Then on submit set the link using this endpoint https://learn.quickpay.net/tech-talk/api/services/#PATCH-account---format-